### PR TITLE
Sidecar updates

### PR DESCRIPTION
**Pull request by panigs7**
# Description
Sidecar updates
Update UBI Image

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/583 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration
Sanity tests:
Operator installed and Powerstore driver running successfully:
image

image

Side car versions used by the driver:
image

### CODE REVIEW

File: operatorconfig/driverconfig/common/default.yaml [hunk 1]
```diff
@@ -2,26 +2,26 @@
...
+  attacher: k8s.gcr.io/sig-storage/csi-attacher:v4.2.0
 
   # "images.provisioner" defines the container images used for the csi provisioner
   # container.
-  provisioner: k8s.gcr.io/sig-storage/csi-provisioner:v3.3.0
+  provisioner: k8s.gcr.io/sig-storage/csi-provisioner:v3.4.0
 
   # "images.snapshotter" defines the container image used for the csi snapshotter
-  snapshotter: k8s.gcr.io/sig-storage/csi-snapshotter:v6.1.0
+  snapshotter: k8s.gcr.io/sig-storage/csi-snapshotter:v6.2.0
```
Comment (rensyct) [inline operatorconfig/driverconfig/common/default.yaml:12]: Should this not be v6.2.1?

Comment (panigs7) [reply operatorconfig/driverconfig/common/default.yaml]: Done

**Review by bharathsreekanth — DISMISSED** [review]

**Comment by rensyct** [comment]
@panigs7, please sign your commits

**Comment by panigs7** [comment]
> @panigs7, please sign your commits

Done

**Review by rajendraindukuri — APPROVED** [review]

**Review by rensyct — APPROVED** [review]

**Pull request merged**